StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -293,6 +300,68 @@ extension EmbeddedPaymentElement: EmbeddedFormViewControllerDelegate {
 }
 
 extension EmbeddedPaymentElement {
+    
+    func _confirm() async -> (result: PaymentSheetResult, deferredIntentConfirmationType: STPAnalyticsClient.DeferredIntentConfirmationType?) {
+        let result: (PaymentSheetResult, STPAnalyticsClient.DeferredIntentConfirmationType?) = await {
+            
+            let authContext: STPAuthenticationContext? = {
+                switch configuration.formSheetAction {
+                case .confirm:
+                    if let formViewController {
+                        return formViewController
+                    }
```

Comment:
nit: seems like we should check to make sure things are on screen before presenting?

```
                    if let formViewController,
                       formViewController.view.window != nil{
                        return formViewController
                    } else if let presentingViewController {
                        return STPAuthenticationContextWrapper(presentingViewController: presentingViewController)
                    }
                    return nil
```

Reply:
Added this, although I can't conceive a scenario where this would occur.